code/datums/components/mirage_border.dm

```diff
@@ -0,0 +1,39 @@
+/datum/component/mirage_border
```

Comment:
This doesn't have too much reason to be a component other than organizational purposes atm but I can think of expansions on this functionality that would benefit from it being a component with access to signals.

I'm considering in the future making the component handle the border teleportation as well for example.